prow/config/config_test.go

```diff
@@ -373,14 +373,22 @@ func CheckBazelbuildSpec(t *testing.T, name string, spec *kube.PodSpec, periodic
 	if spec == nil {
 		return tags
 	}
+	// Tags look something like vDATE-SHA or vDATE-SHA-BAZELVERSION.
+	// We want to match only on the date + sha
+	tagRE := regexp.MustCompile(`^([^-]+-[^-]+)(-[^-]+)?$`)
```

Comment:
we might want to doc this somewhere / add a cross ref note in case the format is changed

Reply:
made this fail if the bazelbuild image tag is using an unexpected format